core/commands/diff.py

```diff
@@ -18,6 +18,17 @@
 from ...common import util
 
 
+if False:
```

Comment:
I might be wrong, but shouldn't this be `MYPY = False; if MYPY:` ? Otherwise, would it still be used correctly by mypy?

Ref: https://github.com/python/mypy/issues/3216#issuecomment-296318286

Reply:
I've seen both and both do work with mypy. If Guido wants your version I can change it but both do look idiosyncratic.

Reply:
Good to know the `if False` works too, agree it's super weird